litestar/handlers/base.py

```diff
@@ -222,6 +270,10 @@ def ownership_layers(self) -> list[Self | Controller | Router]:
 
         return list(reversed(layers))
 
+    @property
+    def app(self) -> Litestar:
```

Comment:
Is this being used anywhere?

Reply:
Yes, I added it to use it. It's used in the resolve DTO properties